Jellyfin.Plugin.NextPVR/Configuration/configPage.html

```diff
@@ -103,6 +109,7 @@
                     page.querySelector('#chkDebugLogging').checked = config.EnableDebugLogging;
                     page.querySelector('#chkNewEpisodes').checked = config.NewEpisodes;
                     $('#selRecDefault', page).val(config.RecordingDefault);
+                    $('#selRecTransport', page).val(config.RecordingTransport);
```

Comment:
jQuery may or may not still be working, this needs testing, and most likely needs replacing by vanilla JS (`querySelector`).
I'd consider that out of scope for this PR however.